core/multivm_deps/vm_virtual_blocks/src/old_vm/oracles/storage.rs

```diff
@@ -46,6 +48,8 @@ pub struct StorageOracle<S: WriteStorage, H: HistoryMode> {
     // to cover this slot.
     // `paid_changes` history is necessary
     pub(crate) paid_changes: HistoryRecorder<HashMap<StorageKey, u32>, HistoryEnabled>,
+
+    initial_values: HistoryRecorder<HashMap<StorageKey, U256>, HistoryEnabled>,
```

Comment:
This is `multivm_deps` folder, while you should modify the `vm` folder (where the latest VM resides)

Reply:
it looks like rebasing main changed the modified file